### PR TITLE
Rough implementation of Moderation Tools

### PR DESCRIPTION
**Pull request by CodyFraker**
Kick, ban, and mute now all have their own commands for mods to be able to use.

### CODE REVIEW

File: Bloon/Features/ModTools/BanUser.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            {
+                // User was not found via ID. Tell mod.
+                await ctx.RespondAsync($"Failed to find a guild member with the user ID of `{discordId}`");
+                await ctx.Message.CreateReactionAsync(DiscordEmoji.FromName(ctx.Client, ":warning:"));
+            }
+        }
+
+        [Command("unban")]
+        [Description("Unbans a user from the SuperbossGames server.")]
+        [Aliases("unyeet")]
```
Comment (dukeofsussex) [inline Bloon/Features/ModTools/BanUser.cs:92]: Why is `unyeet` = unban, but `yeet` = [kick](https://github.com/BloonBot/Bloon/pull/13/commits/9488a9179cbe4a199e177f61dd6524735a733c9a#diff-e71039968905a9ec95f27cdeb6abc8b8cbdcb5537e44f4ee9abafb74ccecd55bR49)?

File: Bloon/Features/ModTools/Models/ModActions.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+namespace Bloon.Features.ModTools
+{
+    public enum Actions
```
Comment (dukeofsussex) [inline Bloon/Features/ModTools/Models/ModActions.cs:3]: Rename to `ModAction`, file as well please

File: Bloon/Features/ModTools/Models/ModEvents.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+namespace Bloon.Features.ModTools
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    [Table("mod_events")]
```
Comment (dukeofsussex) [inline Bloon/Features/ModTools/Models/ModEvents.cs:7]: Singular

File: Bloon/Features/ModTools/Models/ModEvents.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    public class ModEvents
```
Comment (dukeofsussex) [inline Bloon/Features/ModTools/Models/ModEvents.cs:8]: Singular

**Review by dukeofsussex — CHANGES_REQUESTED** [review]
You seem to have tested all commands thoroughly already, so please just fix the warnings and suggestions in VS's error list.

File: Bloon/Features/ModTools/BanUser.cs (hunk 1 above)
Comment (CodyFraker) [reply Bloon/Features/ModTools/BanUser.cs:92]: because yes

File: Bloon/Features/ModTools/Models/ModEvents.cs (hunk 3 above)
Comment (CodyFraker) [reply Bloon/Features/ModTools/Models/ModEvents.cs:7]: w0w yes okay i shall fix this very front end related issue ty for point it out

**Review by dukeofsussex — APPROVED** [review]

**Pull request merged**